src/lhm.net.nuspec

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Lhm.Net</id>
+    <version>0.1.0.0-pre</version>
+    <title>Lhm.Net</title>
+    <authors>Richard Forrest, P G Amila Prabandhika</authors>
+    <owners>Bede Gaming Ltd</owners>
```

Comment:
I guess we can move it into the Bede Gaming Repo then we have the CI process setup as well.